platform/darwin/src/MGLFillExtrusionStyleLayer.h

```diff
@@ -334,6 +334,27 @@ MGL_EXPORT
 
 @property (nonatomic, null_resettable) NSExpression *fillExtrusionTranslateAnchor __attribute__((unavailable("Use fillExtrusionTranslationAnchor instead.")));
 
+/**
+ Whether to apply a vertical gradient to the sides of a fill-extrusion layer. If
+ true, sides will be shaded slightly darker farther down.
+ 
+ The default value of this property is an expression that evaluates to `YES`.
+ Set this property to `nil` to reset it to the default value.
+ 
+ You can set this property to an expression containing any of the following:
+ 
+ * Constant Boolean values
+ * Predefined functions, including mathematical and string operators
+ * Conditional expressions
+ * Variable assignments and references to assigned variables
+ * Step functions applied to the `$zoomLevel` variable
+ 
+ This property does not support applying interpolation functions to the
+ `$zoomLevel` variable or applying interpolation or step functions to feature
+ attributes.
+ */
+@property (nonatomic, null_resettable) NSExpression *fillExtrusionVerticalGradient;
```

Comment:
Paint and layout properties aren’t strongly typed in the iOS and macOS map SDK, so it’s important to conform to the Objective-C and Swift convention that Boolean-typed properties are named in a way that suggests a Boolean value. We’ll want to call this property `fillExtrusionHasVerticalGradient`. Override the property name as `fill-extrusion-has-vertical-gradient`:

https://github.com/mapbox/mapbox-gl-native/blob/ad03e42b2890fd430e99e543a40be42f09f47fba/platform/darwin/scripts/style-spec-cocoa-conventions-v8.json#L33-L36

The change will only affect the runtime styling API, not style JSON parsing. After adding the override, rerun `make darwin-style-code`.